cmake/FindH5Part.cmake

```diff
@@ -1,32 +1,48 @@
-#------------------------------------------------------------------------------#
-# Copyright (c) 2016 Los Alamos National Security, LLC
-# All rights reserved.
-#------------------------------------------------------------------------------#
+################################################################################
+#
+# \file      cmake/FindH5Part.cmake
+# \author    C. Junghans
+# \copyright 2016, Los Alamos National Security, LLC.
+# \brief     Find the H5Part library
+# \date      Mon 05 Dec 2016 09:02:53 AM MST
+#
+################################################################################
```

Comment:
I am never sure what the point of this block is! Isn't `git blame` more accurate?